cd/scripts/release-controller.sh

```diff
@@ -43,7 +43,7 @@ WORKSPACE_DIR=$TEST_INFRA_DIR/..
 SERVICE_CONTROLLER_DIR="$WORKSPACE_DIR/$AWS_SERVICE-controller"
 
 # Check all the dependencies are present in container.
-source "$SCRIPTS_DIR"/lib/common.sh
+source "$TEST_INFRA_DIR"/scripts/lib/common.sh
```

Comment:
necessary?

Reply:
Yeah, Keeping one "common.sh" and reusing for soak & controller-release, instead of having 2 separate.
Both(soak & controller-release) need the same set of functions i.e. check-installed, login-to-buildah-and-helm etc..